runtime/v2/manager.go

```diff
@@ -276,6 +288,79 @@ func (m *ShimManager) startShim(ctx context.Context, bundle *Bundle, id string,
 	return shim, nil
 }
 
+func (m *ShimManager) resolveRuntimePath(runtime string) (string, error) {
+	if runtime == "" {
+		return "", fmt.Errorf("no runtime name")
+	}
+
+	// Custom path to runtime binary
+	if strings.Contains(runtime, "/") {
```

Comment:
How about using filepath#IsAbs instead? Right now names `../../foo/bar` could work. While it wouldn't cause anything harmful, I'd like to make this logic more restricted.

Reply:
agree